junction/templates/profiles/dashboard.html

```diff
@@ -17,7 +17,7 @@
 
 {% block breadcrumbs %}
     {{ block.super }}
-    {% breadcrumb "Dashboard" "user-proposals-list" %}
+    {% breadcrumb "Dashboard" "/profiles" %}
```

Comment:
Same for here, add the comment for second argument.

Reply:
Done.